islands/flow.tsx

```diff
@@ -113,32 +175,77 @@ export default function Flow({ data }: { data: GetServiceMapResponse }) {
     setEdges(newEdges);
   }, [nodes]);
 
+  useEffect(() => {
+    if (nodes) {
+      console.log(edges);
+      console.log("the fuck is this", rfInstance);
+      setStorage();
```

Comment:
Currently saving to local storage but not doing anything with it yet. Not 100% sure how to implement but not sure its that important for this week